QAT/centos-7.4/media/dev/README.md

```diff
@@ -0,0 +1,64 @@
+This docker image is part of Open Visual Cloud software stacks. Optimized for NGINX web server with compute-intensive operations acceleration with Intel® QuickAssist Technology (Intel® QAT).The docker image can be used in the FROM field of a downstream Dockerfile.
+
+## Supported tags and respective Dockerfile links
+ - [qat-centos-74-media-dev](https://github.com/OpenVisualCloud/Dockerfiles/blob/master/QAT/centos-7.4/media/dev/Dockerfile)
+
+## Quick reference
+- #### Supported platform and OS
+Intel&reg; QAT platform, CentOS-7.4.1708
+
+- #### Included components:
+[FFmpeg](https://github.com/OpenVisualCloud/Dockerfiles/blob/master/doc/ffmpeg.md)	[GStreamer](https://github.com/OpenVisualCloud/Dockerfiles/blob/master/doc/gst.md)	
+
+
+- #### Where to get help:
+- [Open Visual Cloud Dockerfiles Github](https://github.com/OpenVisualCloud/Dockerfiles)
+- [Getting Started With Open Visual Cloud Docker Files](https://01.org/openvisualcloud/documents/get-started-docker)
+- [the Docker Community Forums](https://forums.docker.com)
+- [the Docker Community Slack](https://www.docker.com/docker-community)
+- [Stack Overflow](https://stackoverflow.com/questions/tagged/docker)
+
+-  #### Where to file issues:
+[OpenVisualCloud Dockerfiles Issues](https://github.com/OpenVisualCloud/Dockerfiles/issues)
+
+- #### Maintained by:
+[OpenVisualCloud Dockerfiles Community](https://github.com/OpenVisualCloud/Dockerfiles/graphs/contributors)
+
+
+## License
+This docker installs third party components licensed under various open source licenses.  The terms under which those components may be used and distributed can be found with the license document that is provided with those components.  Please familiarize yourself with those terms to ensure your distribution of those components complies with the terms of those licenses.
+
+
+| Components | License |
+| ----- | ----- |
+|CentOS| [Various](https://hub.docker.com/_/centos) |
+|NGINX_HTTP_FLV|BSD 2-clause "Simplified" License|
+|NGINX_Upload_Module|BSD 3-clause "Simplified" License|
+|qat-zip|BSD 3-clause "New" or "Revised" License|
+|openssl|Apache License 2.0|
+|qat-engine|BSD 3-clause "New" or "Revised" License|
+|asynch_mode_nginx |BSD 3-clause "New" or "Revised" License|
+|libogg|BSD 3-clause "New" or "Revised" License|
+|libvorbis|BSD 3-clause "New" or "Revised" License|
+|Opus Interactive Audio Codec|BSD 3-clause "New" or "Revised" License|
+|libvpx|BSD 3-clause "New" or "Revised" License|
+|Aomedia AV1 Codec Library|BSD 2-clause "Simplified" License|
+|x264|GNU General Public License v2.0 or later|
+|x265|GNU General Public License v2.0 or later|
+|dav1d|BSD 2-clause "Simplified" License|
+|Intel SVT-HEVC|BSD-2-Clause Plus Patent License|
+|Intel SVT-AV1|BSD-2-Clause Plus Patent License|
+|Intel SVT-VP9|BSD-2-Clause Plus Patent License|
+|gstreamer|GNU Lesser General Public License v2.1 or later|
+|gst orc|GNU Lesser General Public License v2.1 or later|
```

Comment:
There is no GST in the nginx images.

Reply:
Not sure where to write 'how-to' for this script. Wiki ? OR somwhere in README?
     Add a cmake parameter UPDATE_IMAGE_README, setup "ON" to trigger the script. Then write the 'how-to' to cmake.md

Reply:
It is OK. GST is not included in nginx images.